photonjs/README.md

```diff
@@ -620,6 +623,15 @@ Proposed simpler model:
 
 <img src="./pagination-simplified.png" alt="image-20191229152718492" style="zoom:50%;" />
 
+# Raw database access
+Raw database access is being exposed in a minimal fashion with the `prisma.raw()` method.
+It allows you to execute raw SQL (and later as well different query languages, when NoSQL databases are added) on the database. While this raw database access enables you to work around the limitations of the Prisma Client queries, it's important to note, that this initial version doesn't protect you from SQL injections. So use this with precaution.
+
+## Raw Usage
```

Comment:
We need to have an exhaustive list of all possible output formats. This can be added later if not clear yet.